src/utils/constants.ts

```diff
@@ -112,6 +112,7 @@ export const NETWORK_DEFAULTS = {
       CU_URL: ANT_AO_CU_URL,
       MU_URL: 'https://mu.ao-testnet.xyz',
       SCHEDULER: DEFAULT_SCHEDULER_ID,
+      GRAPHQL_URL: ARWEAVE_GRAPHQL_URL,
```

Comment:
what's the reason we are not deferring to AO defaults for MU, SCHEDULER and GRAPHQL?

Reply:
Mostly for easy of control

Reply:
have we ever used it?

Reply:
I believe its technically used whenever you spawn a process, so for example we may want to point it at goldsky instead of arweave.net. But no I don't think we use it much in normal operations, thats mainly the MU, and CU